test_jsonschema.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023-2024 Ericsson AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import subprocess
+
+import jsonschema
+import pytest
+
+# Set up a console logger
+logger = logging.getLogger("__Logger__")
+console_handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    "%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S%p",
+)
+console_handler.setFormatter(formatter)
+logger.addHandler(console_handler)
+
+
+@pytest.mark.parametrize(
+    "filename",
+    subprocess.check_output(["git", "ls-files", "--exclude-standard", "schemas/*.json"])
+    .decode("utf-8")
+    .splitlines(),
+)
+def test_json_schema(filename):
+    with open(filename) as input_file:
+        event_schema = json.loads(input_file.read())
```

Comment:
Nit: This is fine for these small files, but you're unnecessarily reading the whole file into RAM. This would be the canonical way:
```suggestion
        event_schema = json.load(input_file)
```